config/dataplane/dataplane.yaml

```diff
@@ -28,3 +28,19 @@ spec:
         - name: RUST_LOG
           value: debug
         imagePullPolicy: IfNotPresent
+        startupProbe:
```

Comment:
Note: I found that sometimes a delay of launching a pod causes liveness/readiness failures without startupProbe.
Of course after several seconds, a pod works properly, but I think setting this startupProbe is appropriate.